src/main/java/racingcar/Game.java

```diff
@@ -0,0 +1,53 @@
+package racingcar;
+
+import racingcar.Car;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Game {
+    private final List<Car> cars;
+    private final int attempts;
+
+    // Game 클래스의 생성자
+    public Game(List<String> carNames, int counts) {
+        this.cars = carNames.stream().map(name -> new Car(name)).collect(Collectors.toList());
+        this.attempts = counts;
+    }
+
+    // 게임 시작 메소드
+    public void start() {
+        System.out.println("실행 결과");
+        run();
+    }
+
+    // 게임 진행 메소드
+    private void run() {
+        for(int i = 0; i < attempts; i++) {
+            for(Car car : cars) {
+                car.move();
+            }
+            printStatus();
+        }
+        printWinner();
+    }
+
+    // 게임 과정 출력 메소드
+    private void printStatus() {
+        for(Car car : cars) {
+            System.out.println(car.getStatus());
+        }
+        System.out.println();
+    }
+
+    // 최종 우승자 출력 메소드
+    private void printWinner() {
```

Comment:
```suggestion
    private void printWinner() {
        String winners = cars.stream()
    .filter(car -> car.getPosition() == winnerDistance)
    .map(Car::getName)
    .collect(Collectors.joining(", "));

System.out.println("최종 우승자 : " + winners);
```
이런식으로 코드를 한줄로 최적화할 수 있을 것 같아요!